src/backend/test/unit/pia-intake/pia-intake.service.spec.ts

```diff
@@ -2044,6 +2044,10 @@ describe('PiaIntakeService', () => {
             createdByGuid: user.idir_user_guid,
             status: PiaIntakeStatusEnum.COMPLETE,
           },
+          {
+            isActive: true,
+            status: PiaIntakeStatusEnum.COMPLETE,
```

Comment:
this is only for CPO right? no one else should be seeing completed status?

Reply:
This is one of about 25 scenarios in the test suite. I think comments, as they already exist, are better defined in the actual logic. Writing it here means, we should explain each where clauses in all the rest of massive 4000 liner file. 

You might be feeling out-of-place on this one, perhaps because you're just seeing a snippet, and not the complete file.

Happy to add extra comments in the actual logic, if it's lacking.